docs/tutorial/getting-started.md

```diff
@@ -0,0 +1,22 @@
+# Tutorial: deploy the gateway API integrator charm
+
+This tutorial will walk you through deploying the gateway-api-integrator charm
```

Comment:
I think this statement could be expanded into a proper "What you'll do" section. I imagine there would be two bullet points:

- Deploy the gateway-api-integrator charm
- Integrate with the self-signed-certificates charm [adding a link to the charm if applicable]

Reply:
good point, I added a `what you'll do section`